### ISSUE TITLE
Status of this fork

### ISSUE THREAD

**Issue by georgefst** [georgefst/Euterpea2#1]
See what's left of https://github.com/Euterpea/Euterpea2/pull/52 for the history.

I really don't want to maintain this myself, not least because I don't currently actually have a use case for the library. But, in the short term, I think it's important that an up-to-date version is available _somewhere_. Euterpea is really cool, and I'm sure there are a lot of people who want to use it alongside all the recent improvements in GHC, Cabal and HLS (and when I finally pick my copy of HSoM back up, I'm sure I'll be one of them).

Of course, this fork being not on Hackage, it isn't _trivial_ to use (and I'd really rather not upload to Hackage unless it looks like an upstream solution is unattainable). The easiest way, if using Cabal, is to add to a project's `cabal.project`:
```
source-repository-package
    type: git
    location: https://github.com/georgefst/Euterpea2
    tag: e4f0a23ac31c6565668282e3ca4487ddfca0fbff
```

For the record, while I don't remember the exact contents of the now-deleted messages on the old thread, I genuinely do not understand why they have been deleted (aside, maybe, from the final one, which alluded to the maintainer's lifestyle, but even that was not totally irrelevant, or indeed derogatory). I mention this only because the state of that thread might now imply to a new reader that there was a much more hostile discussion than the one I recall taking place.

**Comment by dschrempf** [comment]
Thank you so much for caring and brining the library up to date! This is really important to keep the project alive. It should be advertized more prominently that there is a maintained version.

**Comment by ninioArtillero** [comment]
Thanks for this effort. I was thinking whether we should gather efforts around a community version of the library to keep it up to date. Perhaps we could then move away from the overriding in `cabal.project` method (which worked great for me!) to a proper hackage package. For the mean time I sent a PR, considering that other like me will eventually reach your fork (I found it on reddit).

**Comment by georgefst** [comment]
> I was thinking whether we should gather efforts around a community version of the library to keep it up to date.

Yes, perhaps. I don't personally have the bandwidth or motivation to get involved.

I would suggest reaching out to Donya first to see whether she'd appreciate the support upstream. At the very least, anyone leading such an effort should try to explain how annoying installing a very old Haskell toolchain is for most people, and understand where the hostility to modernising the package stems from.
